frontend/pages/schedule/ManageSchedulePage/components/NoSchedule/NoSchedule.tsx

```diff
@@ -0,0 +1,359 @@
+/**
+ * Component when there is no schedule set up in fleet
+ */
+import React from "react";
+import { useDispatch } from "react-redux";
+import { push } from "react-router-redux";
+import paths from "router/paths";
+
+// TODO: Look into importing SVG from images directory to clean up clutter
+// import scheduleSvg from "../../../../../assets/images/schedule.svg";
```

Comment:
Can this be done now? I see the svg has been included in the PR.

Reply:
Done! ✅